gulpfile.js

```diff
@@ -770,7 +770,7 @@ var generateTable = function(schema)
 {
     var table = "";
 
-    table += "<table class='table table-bordered table-responsive table-hover table-condensed'>";
+    table += "<table class='table table-responsive table-condensed'>";
```

Comment:
I wanted to remove these classes from the tables generated by alpaca since we don't use them and so that I didn't have to overwrite them every time. I'm not familiar with gulpfiles, though. Is this something that I shouldn't touch?